src/ui/public/chrome/chrome.js

```diff
@@ -7,6 +7,7 @@ import metadata from 'ui/metadata';
 import 'babel/polyfill';
 import $ from 'jquery';
 import 'ui/timefilter';
+import 'ui/notify';
```

Comment:
This registers a bunch of notification stuff with Angular. Originally this was executed as a side-effect of requiring in `/api/tabs` (below), which imported `TabCollection`, which imported `Tab`, which imported `ui/notify`.